src/Makefile.am

```diff
@@ -60,6 +60,8 @@ csmith_SOURCES = \
 	AbsRndNumGenerator.h \
 	ArrayVariable.cpp \
 	ArrayVariable.h \
+	Attribute.cpp \
```

Comment:
You probably forgot to re-generate `Makefile.in` as I advised you.

Reply:
I have added new file in `CMakeLists.txt` and `Makefile.am`. You said `you'all need to run automake to generate Makefile.in`. I didn't get this part. Can you elaborate a bit on how to do it?

Reply:
`marxin@marxinbox:~/Programming/csmith> automake`

Reply:
Done. Should I push it as a separate commit `Updated Makefile` in this pull request?

Reply:
Please include it in this pull request.

Reply:
Done. Please check it.

Reply:
Looks you have an old version of the `automake`, please use this instead: https://pastebin.com/LCXwXzaR and do not make it a new commit, include it where you added `Attribute.cpp` file.

Reply:
Thanks. Done.